src/client/editor/Editor.js

```diff
@@ -180,7 +188,7 @@ export default class Editor {
       if (previousURI.endsWith(".gltf")) {
         const name = last(poppedURI.split("/"));
         this._prefabBeingEdited.name = name;
-        this._duplicateNameCounters.set(name, this._duplicateNameCounters.get(name) || 0);
+        this._conflictHandler.addToDuplicateNameCounters(name);
```

Comment:
It should also de-duplicate the name here. Otherwise this leads to a bug:
1. Load Spoke
1. Add a Node. Name it "test.scene"
1. Drop in a GLTF model
1. "Edit Prefab" on the model
1. Save the prefab as "test.scene"
1. Return to the unsaved scene
1. There are now two "test.scene" nodes.